index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body style="height: 100%">
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <nav id="navbar">
+      <a href="#home">Home</a>
+      <a href="#work">Work</a>
+      <a href="#contact">Contact</a>
+      <a href="#about">About</a>
+    </nav>
+
+    <section id="welcome-section" style="height: 100%">
+      <h1>Welcome to my Portfolio</h1>
+    </section>
```

Comment:
Nice job 👍 mentioning the CSS should be in a separate file. But I'm not sure about the semantic tag to use for the header because it's a good practice used in the code to separate the main title with the `section` tag and its proper use of `h1` for the main title of the page. Kindly remove the first comment about the semantic tag of the header.